recipe/patch_yaml/mp-api.yaml

```diff
@@ -0,0 +1,6 @@
+if:
+  artifact_in: mp-api-0.37.1-pyhd8ed1ab_0.conda
```

Comment:
The issue was fixed in https://github.com/conda-forge/mp-api-feedstock/pull/50 so only a single release is affected.